### PR TITLE
Require regular key rollover

### PR DESCRIPTION
**Pull request by KellerFuchs**
This is probably overly careful, but easy to implement.

### CODE REVIEW

**Comment by KellerFuchs** [comment]
@hashbang/administrators Ping?

**Comment by daurnimator** [comment]
Are we really concerned about less-that-perfect random numbers? If we don't trust /dev/random we have bigger issues.

**Comment by KellerFuchs** [comment]
@daurnimator It's more about recovering automatically from a key compromise (either due to a vuln or due to us messing up the crypto).
However, I totally forgot a paragraph between “less-than-perfect RNG” and “To mitigate this”  ⭕ 

Good catch!

File: abuse/signup.md [hunk 1]
```diff
@@ -123,6 +124,37 @@ rollover procedures and secure key storage.
...
+Both are comfortably over what can be expected to be served by the API server
+over its lifetime (i.e. without restart).  However, less-than-perfect random
+number generation can significantly increase the risk of accidental nonce reuse.
+
+More importantly, a variety of other factors can lead to key compromise:
+vulnerabilities in our own code, flawed crypto implementation, ...
+
+To mitigate this, the implementation should enforce that the encryption key is
+volatile, by replacing it with a new random one, whenever a new `/captcha`
+request occurs and either of those conditions are met:
```
Comment (lrvick) [inline abuse/signup.md]: these?

Comment (lrvick) [inline abuse/signup.md]: these?

Comment (KellerFuchs) [reply abuse/signup.md]: Both seem legit.
No strong feeling about it, I can break out the _Chicago Manual of Style_ when I'm back...  :P

Comment (KellerFuchs) [reply abuse/signup.md]: I totally forgot about that, but it seems that “these” is more correct, due to the lack of distance in the text.

**Comment by KellerFuchs** [comment]
Were there any other issues?
I just fixed lrvick's comment and rebased on the current master.

**Pull request merged**